src/components/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button } from '@chakra-ui/react';
+
+type ButtonProps = {
```

Comment:
For prop types, let's go with using interfaces over type aliases. The [Typescript doc ](https://docs.google.com/document/d/13pDems-Xki2M7OF5dgpTVojtEQMwt4ZTsuDn4iqtfrQ/edit?usp=sharing) I linked in the dev channel goes more into detail, but generally we want to use interfaces to describe objects (like props) and type aliases for unions/intersections/etc.